util/src/macros.rs

```diff
@@ -51,3 +51,21 @@ macro_rules! unreachable_unchecked {
         }
     });
 }
+
+/// Variadic version of [`core::cmp::max`].
+#[macro_export]
+macro_rules! max {
+    ($arg:expr) => { $arg };
+    ($arg1:expr, $($arg:expr),+) => {
+        core::cmp::max($arg1, $crate::max!( $($arg),+ ))
+    };
+}
+
+/// Variadic version of [`core::cmp::min`].
+#[macro_export]
+macro_rules! min {
+    ($arg:expr) => { $arg };
+    ($arg1:expr, $($arg:expr),+) => {
+        core::cmp::min($arg1, $crate::min!( $($arg),+ ))
+    };
+}
```

Comment:
I think these are unused now. Do we want to keep them around?

Reply:
imo they're nice to have but i can nuke them